botorch/utils/gp_sampling.py

```diff
@@ -178,11 +178,13 @@ def _get_weights(
             num_rff_features: The number of fourier features.
             sample_shape: The sample shape of weights.
         Returns:
-            A `(sample_shape) x input_dim x num_rff_features`-dim tensor of weights
+            A `(sample_shape) x (kernel_batch_shape) x input_dim x num_rff_features`-dim
+            tensor of weights.
```

Comment:
```suggestion
            A tensor of weights with shape
            `(*sample_shape, *kernel_batch_shape, input_dim, num_rff_features)`.
```

Are the parentheses supposed to represent that `sample_shape` and `kernel_batch_shape` may not be present, or do they indicate that these might be tuples?

Also technically this is the shape, not the dimension (the dimension is >=2).

Reply:
> Are the parentheses supposed to represent that sample_shape and kernel_batch_shape may not be present, or do they indicate that these might be tuples?

They may be tuples, but we use parentheses are to indicate that they may not be present (or equivalently be empty tuples).